ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -68,8 +72,22 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
   timeout: 2h30m0s
-- always_run: false
-  as: conc-builds-3nodes
+- as: hybrid-control-plane-12nodes
```

Comment:
@paigerube14 Why 12 nodes are used? Is there a limitation on the node number?

Reply:
no limitation, was trying to pick a lower number that we could see the rhel and rchos nodes